Client/Frontend/Reader/ReaderMode.js

```diff
@@ -143,42 +139,52 @@ var _firefox_ReaderMode = {
                 }
             }
         }
-    },
+    }
 
-    showContent: function() {
+    function showContent() {
         // Make the reader visible
         var messageElement = document.getElementById('reader-message');
         messageElement.style.display = "none";
         var headerElement = document.getElementById('reader-header');
         headerElement.style.display = "block"
         var contentElement = document.getElementById('reader-content');
         contentElement.style.display = "block";
-    },
-    
-    configureReader: function() {
+    }
+
+    function configureReader() {
         // Configure the reader with the initial style that was injected in the page.
         var style = JSON.parse(document.body.getAttribute("data-readerStyle"));
-        _firefox_ReaderMode.setStyle(style);
+        setStyle(style);
 
         // The order here is important. Because updateImageMargins depends on contentElement.offsetWidth which
         // will not be set until contentElement is visible. If this leads to annoying content reflowing then we
-        // need to look at an alternative way to do this.
-        _firefox_ReaderMode.showContent();
-        _firefox_ReaderMode.updateImageMargins();
+        // need to look at an alternative way to do 
+        showContent();
+        updateImageMargins();
     }
-};
 
-window.addEventListener('load', function(event) {
-    // If this is an about:reader page that we are loading, apply the initial style to the page.
-    if (document.location.href.match(/^http:\/\/localhost:\d+\/reader-mode\/page/)) {
-        _firefox_ReaderMode.configureReader();
-    }
-});
+    window.__firefox__.reader = {
+        // If this is http or https content, and not an index page, then try to run Readability. If anything
+        // fails, the app will never be notified and we don't show the button. That is ok for now since there
+        // is no error feedback possible anyway.
+        DEBUG: false,
+        checkReadability: checkReadability,
+        readerize: readerize,
+        setStyle: setStyle
+    };
+
+    window.addEventListener('load', function(event) {
+        // If this is an about:reader page that we are loading, apply the initial style to the page.
+        if (document.location.href.match(/^http:\/\/localhost:\d+\/reader-mode\/page/)) {
```

Comment:
While here, perhaps extract this ugly regex into a helper so we don't have to duplicate it.